config.ru

```diff
@@ -20,16 +20,24 @@ end
 
 if log_path
   enable :logging
-  log = File.new(log_path, "a")
-  log.sync = true
-  STDOUT.reopen(log)
-  STDERR.reopen(log)
+  unless ENV["LOG_TO_STDOUT"].present?
```

Comment:
I find `unless ENV["LOG_TO_STDOUT"].present?` a little confusing, is `if !ENV.key?("LOG_TO_STDOUT")` clearer?

Reply:
I dislike unless, but since it's the style we'd best roll with it.